tox.ini

```diff
@@ -92,7 +92,7 @@ commands =
     python setup.py test --xunit-output=xunit-results {posargs}
 
 [testenv:synchro37]
-whitelist_externals =
+allowlist_externals =
```

Comment:
Changed in tox 3.18. See: https://tox.readthedocs.io/en/latest/config.html#conf-allowlist_externals